docs/upgrading.rst

```diff
@@ -17,6 +17,35 @@ The marshmallow 3.x series supports Python 2.7, 3.4, 3.5, and 3.6.
 
 Python 2.6 and 3.3 are no longer supported.
 
+
+``strict`` in Schema is deprecated
+**********************************
+
+`Schema().load` and `Schema().dump` don't return a (data, errors) duple anymore.
+
+Only data is returned. If invalid data are passed in, a :exc:`ValidationError <marshmallow.exceptions.ValidationError>` is raised. The dictionary of validation errors is accessible from the `ValidationError.messages <marshmallow.exceptions.ValidationError.messages>` attribute, along with the valid data (values deserialized without error).
+
+.. code-block:: python
+
+    from marshmallow import Schema, ValidationError
+
+    # 2.x
+    user = User(name="Monty", email="monty@python.org")
+    schema = UserSchema()
+    data, errors = schema.dump(user)
+
+    # 3.x
+    user = User(name="Monty", email="monty@python.org")
+    schema = UserSchema()
+    try:
+        data = schema.dump(user)
+    except ValidationError as err:
+        errors = err.messages
+        valid_data = err.valid_data
+
+:meth:`Schema.validate` now always returns a dictionary of validation errors (same as before with ``strict=True``).
```

Comment:
Wouldn't be the same as before with `strict=False`?

Reply:
Oops. You're right. Let me fix this now.